### PR TITLE
layout component

### LINKED ISSUE #56: Layout Component
Layout component to wrap all pages that need access to the header.

### PR DESCRIPTION
**Pull request by ctc-devops**
Layout component to wrap all pages that need access to the header.
closes #56

### CODE REVIEW

File: src/components/NavBar/NavBar.jsx [hunk 1]
```diff
@@ -1,34 +1,50 @@
...
+);
+
+const NavBar = ({ isAdmin }) => {
   return (
-    <div className="nav-bar">
-      <div className="fya-logo">
+    <div className={styles['nav-bar']}>
+      <div className={styles['fya-logo']}>
         <img src={FYALogo} alt="Find Your Anchor Logo" />
       </div>
```
Comment (meghakak) [inline src/components/NavBar/NavBar.jsx]: Could we add a Link tag for this image to link to the home page ('/') when clicked?

**Review by meghakak — CHANGES_REQUESTED** [review]
Just one small change for the navbar! Thank you for picking up this task :)

**Review by meghakak — APPROVED** [review]
LGTM

**Review by sylvesteryue — APPROVED** [review]
LGTM

**Pull request merged**